packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -50,7 +44,7 @@ async function compileFixture({ input, dirname }: { input: string; dirname: stri
 
     await bundle.write({
         file: outputFile,
-        format: 'cjs',
+        format: 'esm',
```

Comment:
Jest doesn't like ES Modules and Vitest doesn't like CJS... Hopefully this is fine.